app/components/forms/admin/settings/ticket-fees-form.js

```diff
@@ -5,6 +5,7 @@ const { Component } = Ember;
 export default Component.extend({
   actions: {
     submit() {
+      this.sendAction('save');
```

Comment:
use `save` action directly in the template. No need this intermediate `submit` action here.